CDCgov/prime-reportstream
# 15310: add index on report_id in action_log

This PR adds an index on report_id in the action_log table

Test Steps:
1. Run `./gradlew quickRun`
2. Verify that there is an index called action_log_report_id_idx in action_log on the report_id column

## Changes
- adds an index on report_id in the action_log table

## Checklist

### Testing

### Process

## Linked Issues
- #15310

## To Be Done


## Specific Security-related subjects a reviewer should pay specific attention to
